api/openapi-spec/swagger.json

```diff
@@ -46070,35 +46070,6 @@
       "required": true
      }
     ]
-   },
-   "/version/": {
```

Comment:
I see. I am going back to my previous question then. That means those subpaths are not registered with a webservice and does not have Swagger/OpenAPI spec? Is it not a better idea to just register them with a webservice?